src/components/posts/PostList.js

```diff
@@ -8,7 +8,7 @@ import CategoryDropDown from "./CategoryDropDown"
 export const PostList = props => {
     const { getPosts, posts } = useContext(PostContext)
 
-    posts.sort((a,b) => b.creation_time - a.creation_time)
+    posts.sort((a,b) => b.publication_date - a.publication_date)
```

Comment:
My suspicion is that this doesn't work, though I haven't done extensive testing on it yet... Our `publication_date` property on a post is now a string in "YYYY-MM-DD" format, not a timestamp... we may want to convert these to timestamps in this sort method maybe?